README.md

```diff
@@ -121,6 +122,93 @@ In order to test the installation routine for promptMastermind, we use a set of
 In addition, [circleci][circleci-uri] is used to build all platforms, including macOS as part of our pull request
 validation. Special thanks to [circleci][circleci-uri] for providing the build agents!
 
+## Features
+
+### Two-Line Prompt
+
+The first-line displays username, hostname, and current path.
+
+The second-line emits information about the current branch and status when in a git path. If using [pyenv][pyenv-url] and [direnv][direnv-url] the second-line also displays the currently activated Python virtualenv.
+
+### SDK Installs
+
+[AWS SDK][aws-sdk-url]
+
+```sh
+aws-install
+```
+
+[Google Cloud SDK][gcloud-sdk-url]
+
+```sh
+gcloud-install
+```
+
+[.Net SDK][dotnet-sdk-url]
+
+```sh
+dotnet-install
+```
+
+### Colors
+
+Global colors defined by promotMastermind:
+
+| Name               | Color                   |
+| ------------------ | ----------------------- |
+| CLR_CLEAR          | DEFAULT COLOR           |
+| CLR_BLACK          | ANSI BLACK (GRAY)       |
+| CLR_RED            | ANSI RED                |
+| CLR_GREEN          | ANSI GREEN              |
+| CLR_YELLOW         | ANSI YELLOW             |
+| CLR_BLUE           | ANSI BLUE               |
+| CLR_MAGENTA        | ANSI MAGENTA (PURPLE)   |
+| CLR_CYAN           | ANSI CYAN               |
+| CLR_WHITE          | ANSI WHITE              |
+| CLR_BRIGHT_BLACK   | BRIGHT BLACK (GRAY)     |
+| CLR_BRIGHT_RED     | BRIGHT RED              |
+| CLR_BRIGHT_GREEN   | BRIGHT GREEN            |
+| CLR_BRIGHT_YELLOW  | BRIGHT YELLOW           |
+| CLR_BRIGHT_BLUE    | BRIGHT BLUE             |
+| CLR_BRIGHT_MAGENTA | BRIGHT MAGENTA (PURPLE) |
+| CLR_BRIGHT_CYAN    | BRIGHT CYAN             |
+| CLR_BRIGHT_WHITE   | BRIGHT WHITE            |
+
+### macOS Commands
+
+Show/Hide Files:
+
+```sh
+# show all hidden files
+show-all
+
+# hide all hidden files
+hide-all
+```
+
+Flush DNS:
+
+```she
```

Comment:
typo: should be "sh" -- also, flush-dns is macOS only -- might want to say that. :)

Reply:
nvm; you did! lol